Gems/Multiplayer/Code/Source/Components/NetBindComponent.cpp

```diff
@@ -260,6 +260,45 @@ namespace Multiplayer
         return m_netEntityMigration;
     }
 
+    bool NetBindComponent::ValidatePropertyRead(const char* propertyName, NetEntityRole replicateFrom, NetEntityRole replicateTo) const
+    {
+        bool isValid(false);
+        if (replicateFrom == NetEntityRole::Authority)
+        {
+            // Things that replicate to clients are readable by all network entity roles
+            const bool replicatesToClient = (replicateTo == NetEntityRole::Client);
+            // Things that replicate from Authority can be read by all hosts
+            const bool isHost = (IsNetEntityRoleAuthority() || IsNetEntityRoleServer());
+            // Things that replicate to Autonomous can't be read by clients
+            const bool isAutonomous = ((replicateTo == NetEntityRole::Autonomous) && !IsNetEntityRoleClient());
+            isValid = replicatesToClient || isHost || isAutonomous;
+        }
+        else
+        {
+            // Autonomous can only replicate to Authority, and won't replicate to servers, it's meant for client authoritative values like basic client metrics
```

Comment:
Is Autonomous->Auth properties also useful for game affecting logic besides basic metrics? For example, if I want to change my mouse sensitivity, the server would need to know about that so that my aiming angles are updated properly? Or is that an RPC thing?

Reply:
Technically you'd want to put such data into your input, since the values modify how the inputs are processed.. or you could expect the client to pre-bake any acceleration curves or damping directly into the aim delta values that are transmitted to the server.